src/views/tools/image_management.test.ts

```diff
@@ -67,24 +111,26 @@ test("Downloads card images, resizes, and stores them.", async () => {
     allMissingCards: false,
     cardIds: ["1", "2", "3"],
   });
-  await timeout(1000);
+
+  // Wait for the image update to finish.
+  await timeout(2000);
```

Comment:
Is there a way to await the update itself rather than an arbitrary timeout value? If so, the function could potentially run a little faster.

Reply:
Great point, it's definitely worth the effort to make this run faster.